src/AggregateFunctions/AggregateFunctionGroupArraySorted.h

```diff
@@ -0,0 +1,355 @@
+#pragma once
```

Comment:
In the meantime the style changed. No more `.h` for aggregations unless they are necessary (and it isn't in this case)

Reply:
Updated.